deps/build.jl

```diff
@@ -1,41 +1,188 @@
+using CUDAdrv
```

Comment:
Running Pkg.build("CUDArt") seems to force building of CUDAdrv everytime altought not a big deal...kinda annoying (perhaps this is done for safety reasons)